src/panels/githubissues.js

```diff
@@ -0,0 +1,79 @@
+var React = require('react');
+var $ = require('jquery');
+var classNames = require('classnames');
+
+/**
+ * Github Issues panel.
+ */
+var IssuesPanel = React.createClass({
+    getInitialState: function() {
+        return {
+            issues: []
+        };
+    },
+
+    componentDidMount: function() {
+        $.getJSON('https://api.github.com/repos/IllinoisWCS/IllinoisWCS.github.io/issues')
+            .then((data) => {
+                this.setState({
+                    issues: data
+                })
+            });
+        this.updateData();
+    },
+
+    updateData: function() {
+        $.getJSON('https://api.github.com/repos/IllinoisWCS/IllinoisWCS.github.io/issues')
+            .then((data) => {
+                this.setState({
+                    issues: data
+                })
+            });
+        setTimeout(this.updateData, 12 * 60 * 60 * 1000)
```

Comment:
Missing semicolon. Additionally, I recommend using `setInterval` to match how the other panels handle polling ([example](https://github.com/IllinoisWCS/wcs-display/blob/master/src/panels/mtd.js#L20)).